fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneRepartitionPlanner.java

```diff
@@ -74,6 +74,11 @@ RepartitioningContext determineRepartitioningAction(@Nonnull final Tuple groupin
 
         // candidate partition's current doc count
         final int currentPartitionCount = candidatePartition.getCount();
+        if ((currentPartitionCount == 0) && ((olderPartition != null) || newerPartition != null)) {
```

Comment:
There is a `LuceneRepartitionPlannerTest`, it would be good to add some coverage of this behavior there.

Reply:
done.